File: Service_eHospital_DongNai_A/Implement/Transfer_VienPhi_ChiTietService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class Transfer_VienPhi_ChiTietService : BaseService<Transfer_VienPhi_ChiTiet, ITransfer_VienPhi_ChiTietRepository>
    , ITransfer_VienPhi_ChiTietService
    {
    private readonly ITransfer_VienPhi_ChiTietRepository _Transfer_VienPhi_ChiTietRepository;
    public Transfer_VienPhi_ChiTietService(ITransfer_VienPhi_ChiTietRepository Transfer_VienPhi_ChiTietRepository) : base(Transfer_VienPhi_ChiTietRepository)
    {
    _Transfer_VienPhi_ChiTietRepository = Transfer_VienPhi_ChiTietRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/Transfer_VienPhi_ChiTiet.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Transfer_VienPhi_ChiTiet } from './Transfer_VienPhi_ChiTiet.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Transfer_VienPhi_ChiTietService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'Transfer_VienPhi_Id', 'BenhNhan_Id', 'NgayChungTu', 'IDRef', 'Loai_IDRef', 'LoaiNoiDung', 'NoiDung_Id', 'SoLuong', 'ThanhTien', 'MaDoiTuong', 'VAT'];

    List: Transfer_VienPhi_ChiTiet[] | undefined;
    ListFilter: Transfer_VienPhi_ChiTiet[] | undefined;
    FormData!: Transfer_VienPhi_ChiTiet;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Transfer_VienPhi_ChiTiet";
    }
}

